src/rendering/SceneManager.cc~

```diff
@@ -0,0 +1,2338 @@
+/*
```

Comment:
remove this file

Reply:
Sorry, I dont know why but due to something it comes back...